pkg/ebpf/testdata/run_latency_bench.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [ "${EUID}" != "0" ]
+then
+  echo "Error: this command must be run as root" >&2
+  exit 1
+fi
+
+if ! [ -x "$(command -v benchstat)" ]
+then
+  echo "Error: benchstat could not be found" >&2
+  exit 1
+fi
+
+now=$(date -u +"%Y-%m-%dT%H%M%S")
+RESFILE=/tmp/ebpf-bench-results.${now}
+STATFILE=/tmp/ebpf-bench-benchstat.${now}
+
+# using the test.count flag doesn't let us report probe times for each iteration
+# use bash loop instead
+for _ in {1..5}
+do
+  go test \
+    -mod=vendor \
+    -tags linux_bpf,ebpf_bindata \
```

Comment:
Do we need to use ebpf_bindata ? I would prefer that we not commit further to having two types of builds just for testing if we can avoid it.

Reply:
We do for now. It is not possible to run tests locally without the `ebpf_bindata` tag.